internal/cli/command/controlplane/exec.go

```diff
@@ -188,6 +198,53 @@ func pullImage(options *cpContext, _ cli.Cli) error {
 	return cmd.Run()
 }
 
+func fetchContainerCommandOutputAndError(options *cpContext, cmd []string, cmdEnv map[string]string) string {
+	out, err := fetchContainerCommandOutput(options, cmd, cmdEnv)
+	if err != nil {
+		out += "\n\n" + err.Error()
+	}
+	return out
+}
```

Comment:
**Optional**: I think this **MAY** be written in a more generic fashion such as:
```suggestion
func combineCommandOutput(output string, err error) string { // Note: the input could be []bytes as well, depends on the use-case.
	return output + "\n\n" + err.Error()
}
```